static/src/javascripts/projects/common/modules/identity/upsell/consent-card/FollowCard.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import React, { Component } from 'preact-compat';
```

Comment:
This file was renamed, and modified quite a bit... GitHub thinks it's a new file.